pkg/cmd/experimental/bundlesecret/bundle-secret.go

```diff
@@ -0,0 +1,114 @@
+package bundlesecret
+
+//export OPENSHIFTCONFIG=./openshift.local.certificates/admin/.kubeconfig
+import (
+	"fmt"
```

Comment:
Go style convention:

organize your imports for standard packages to be at top, and separate standard from non standard imports by a line-break

import (
 "fmt"
 "io"
 ...

kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api
...
)